backend/server/src/main/java/ai/verta/modeldb/experiment/FutureExperimentDAO.java

```diff
@@ -228,20 +235,16 @@ public InternalFuture<FindExperiments.Response> findExperiments(FindExperiments
 
                                     return query
                                         .map(
-                                            (rs, ctx) -> {
-                                              var runBuilder =
-                                                  Experiment.newBuilder()
-                                                      .setId(rs.getString("id"))
-                                                      .setProjectId(rs.getString("project_id"))
-                                                      .setName(rs.getString("name"))
-                                                      .setDescription(rs.getString("description"))
-                                                      .setDateUpdated(rs.getLong("date_updated"))
-                                                      .setDateCreated(rs.getLong("date_created"))
-                                                      .setOwner(rs.getString("owner"))
-                                                      .setVersionNumber(
-                                                          rs.getLong("version_number"));
-                                              return runBuilder;
-                                            })
+                                            (rs, ctx) ->
+                                                Experiment.newBuilder()
+                                                    .setId(rs.getString("id"))
+                                                    .setProjectId(rs.getString("project_id"))
+                                                    .setName(rs.getString("name"))
+                                                    .setDescription(rs.getString(DESCRIPTION_FIELD))
```

Comment:
this doesn't seem like a good change. Makes it harder when there's a mix of constants and plain Strings to see what's going on. Recommend reverting.